test/query_component_test.dart

```diff
@@ -60,9 +61,9 @@ class QueryTestComponent {}
 void main() {
   group('In $QueryComponent,', () {
     const emptyEntry = '';
-    const advertiserId = 'advertiser-id';
-    const mediaPlanIdA = 'media-plan-id-a';
-    const ioIdA = 'io-id-a';
+    const advertiserId = '10000';
+    const mediaPlanIdA = '20000';
```

Comment:
opt: we typically use number e.g. mediaPlanId1

Reply:
Since i check for the input format, having non-integer ids will actually disable the button